portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,12 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.link {
+    text-decoration: none;
+}
+
+.icon {
```

Comment:
Something to keep in mind - not necessarily needed for this change because you only have one icon, but in general it's preferred to utilize naming conventions so that there is little or no ambiguity - https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Naming.

Reply:
Thing is, I have two icons, one for github and one for linkedin, that's why I thought about going for the icon class

Reply:
Oh yeah I see now, this is totally fine then!